library/src/main/java/eightbitlab/com/blurview/BlurView.java

```diff
@@ -77,16 +81,48 @@ protected void onAttachedToWindow() {
         }
     }
 
+    /**
+     * @see #setupWith(ViewGroup, BlurAlgorithm, float)
+     */
+    public BlurViewFacade setupWith(@NonNull ViewGroup rootView) {
+        return this.setupWith(rootView, BlurController.DEFAULT_SCALE_FACTOR);
+    }
+
+    /**
+     * @see #setupWith(ViewGroup, BlurAlgorithm, float)
+     */
+    @SuppressWarnings("deprecation")
+    public BlurViewFacade setupWith(@NonNull ViewGroup rootView, float scaleFactor) {
```

Comment:
Instead of adding the scale factor here, you can just override the `scaleFactor` in `BlurAlgorithm`.
Since `RenderScriptBlur` is `final`, you can have a custom algorithm delegating everything on `RenderScriptBlur`, but with a custom `scaleFactor`. 

Although I would advise against it, because the performance with a low scale factor might be terrible on weaker devices, and developers often just test on a couple of high-end devices, so the problem is easy to miss